manim/mobject/graphing/functions.py

```diff
@@ -31,6 +31,9 @@ class ParametricFunction(VMobject, metaclass=ConvertToOpenGL):
     use_smoothing
         Whether to interpolate between the points of the function after they have been created.
         (Will have odd behaviour with a low number of points)
+    use_vectorized
+        Whether to pass in the generated t value array to the function as ``[t_0,t_1,...]``. Only use this if your function supports it.
+        Output should be a numpy array of shape ``[[x_0,x_1,...],[y_0,y_1,...],[z_0,z_1,...]]`` but z can also be 0 if the Axes is 2D
```

Comment:
Lines are a bit long, breaking:

```suggestion
        Whether to pass in the generated t value array to the function as ``[t_0, t_1, ...]``.
        Only use this if your function supports it. Output should be a numpy array
        of shape ``[[x_0, x_1, ...], [y_0, y_1, ...], [z_0, z_1, ...]]`` but ``z`` can
        also be 0 if the Axes is 2D
```